packages/system/src/styles/colors.ts

```diff
@@ -1,12 +1,12 @@
 import * as CSS from 'csstype'
 import { themeGetter } from '../style'
-import { ThemeNamespaceValue, ITheme, Theme } from '../types'
+import { SystemProp, ThemeNamespaceValue, ITheme, Theme } from '../types'
 
 export type ThemeColor<T extends ITheme = Theme> = ThemeNamespaceValue<
   'colors',
   T
 >
-export type Color<T extends ITheme = Theme> = ThemeColor<T> | CSS.Property.Color
+export type Color<T extends ITheme = Theme> = SystemProp<ThemeColor<T> | CSS.Property.Color, T>;
```

Comment:
This changes fixes type acquisition for patterns like:

```ts
<x.div color={{_: 'themeColor'}}
```

Reply:
This is wrong, it has to be fixed here: https://github.com/gregberge/xstyled/blob/main/packages/system/src/styles/typography.ts#L113